openpype/hosts/maya/plugins/publish/extract_thumbnail.py

```diff
@@ -153,7 +153,18 @@ def process(self, instance):
                     )
                 )
 
-            path = capture.capture(**preset)
+            if "reloadTextures" in preset["viewport_options"]:
+                lib.reload_all_udim_tile_previews()
+
+            preset["viewport_options"].pop("reloadTextures", None)
+            if "textures" in preset["viewport_options"]:
```

Comment:
Even when "textures" is disabled it's still in the preset viewport options, but just as False. Right? So this would always evaluate as true - the if statement is wrong.